CMakeLists.txt

```diff
@@ -115,6 +115,7 @@ include_directories(
     # system includes
     deps/outcome
     deps/tinycbor/src
+    deps/libsecp256k1/include
```

Comment:
in theory, modules should have exported their include paths so making it possible for dependants to find their inludes.
it should work except for some "special cases". probably it doesn't work with git submodules?